src/lib/components/InputChip/InputChip.svelte

```diff
@@ -0,0 +1,116 @@
+<script lang="ts">
+	import { createEventDispatcher } from 'svelte/internal';
+
+	// Event Dispatcher
+	const dispatch = createEventDispatcher();
+
+	// Props (settings)
+	/** Optionally provide an semantic label. */
+	export let label: string = '';
+	/** Optionally provide an input placeholder. */
+	export let value: string[] = [];
+	/**
+	 * Provide a whitelist of accepted values.
+	 * @type {string[]}
+	 */
+	export let whitelist: string[] = [];
+	/** When enabled, will format entered values as lowercase. */
+	export let lowercase: boolean = true;
+	/**
+	 * Provide a custom validation function.
+	 * @type {function}
+	 */
+	export let validation: any = undefined;
+
+	// Props (styles)
+	/** Provide chip styles. */
+	export let chip: string = `chip-primary`;
+	/** Provide chip hover styles. */
+	export let hover: string = 'hover:chip-error';
+	/** Set the input padding styles. */
+	export let padding: string = 'p-3';
+	/** Set the input border radius styles. */
+	export let rounded: string = 'rounded-container-token';
+	/** Set the input focus ring styles. */
+	export let ring: string = 'focus:ring-transparent';
+
```

Comment:
feature request: `allowDuplicate` or something similar to allow foot shooting and/or triple choc icecream selection

Reply:
Remember to change the docs if ryceg gets his chocolate, chocolate, chocolate ;)